lib/generators/good_job/install_generator.rb

```diff
@@ -21,6 +21,12 @@ def create_migration_file
       migration_template 'migrations/create_good_jobs.rb.erb', File.join(db_migrate_path, "create_good_jobs.rb")
     end
 
+    def copy_empty_partials
+      %w[custom_job_details custom_execution_details].each do |partial|
+        copy_file "views/_#{partial}.html.erb", "app/views/good_job/jobs/_#{partial}.html.erb"
```

Comment:
I think I'd prefer not to copy these over as part of the install generator. Initially I was excited about it, but I try to step lightly inside of other people's applications and I think I'd prefer to document it in the readme, but otherwise not force it on an application.

Reply:
This also makes me think that the path should either be:

- `views/good_job/_custom_job_details.html.erb` OR
- `views/good_job/jobs/_job_details.html.erb`

...because I think the custom is useful if it's at a simple root path, but otherwise it should just be "override this existing vanilla-looking path"